cqrs-user-command-service/src/main/java/dev/amir/usercommand/domain/exception/DuplicateUserException.java

```diff
@@ -0,0 +1,14 @@
+package dev.amir.usercommand.domain.exception;
+
+import dev.amir.usercommand.domain.valueobject.UserEmail;
+import dev.amir.usercommand.domain.valueobject.UserUsername;
+
+public class DuplicateUserException extends UserException {
+    public DuplicateUserException(UserEmail email) {
+        super("User with email: " + email + " already exists");
```

Comment:
in order to be consistent with the other message in line 12
change the message: `"The email : " + email + " already exists"`